v3.0/how-to/maintain/identify-slow-queries.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: 慢查询日志
 category: how-to
-aliases: ['/docs-cn/sql/slow-query/']
```

Comment:
why remove this?

Reply:
Sorry, action by mistake. Add back in https://github.com/pingcap/docs-cn/pull/1495/commits/8ea96d204e58ed7a18ca67875b63d679ffba950e